src/respo_alerts/comp/alerts.cljs

```diff
@@ -104,8 +128,8 @@
          {}
          (input
           {:style (merge ui/input {:width "100%"}),
+           :class-name schema/input-box-name,
            :placeholder "",
-           :autofocus true,
```

Comment:
Why not?

Reply:
I added `(.select target)` somewhere in the code, which is a replacement for `autofocus`. Plus `autofocus` does not alway work. Not sure why it did work. Maybe caused by timing...